src/Symfony/Component/Notifier/Bridge/Firebase/README.md

```diff
@@ -3,6 +3,14 @@ Firebase Notifier
 
 Provides Firebase integration for Symfony Notifier.
 
+DSN should be as follow:
+
+```
+firebase://username:password@default
```

Comment:
```suggestion
firebase://USERNAME:PASSWORD@default
```

imho uppercased is better to be similar to others doc
and also we clearly see what is "configurable" from what is not :)